migrations/20230516033000_change_error_codes_format.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+SET search_path = devices_api, public;
+
+ALTER TABLE devices_api.error_code_queries ADD column codes_query_response jsonb;
```

Comment:
We have now `query_response` and `codes_query_response`. That's confusing—do you have plans for the older text column?

Reply:
I'm leaving it there since its already deployed and the column probably already holds some data

Reply:
Well we’re pre-release, I think it’s okay to run around deleting.

Reply:
ok

Reply:
Eventually I'd like to make this non-nullable, but I understand the hesitancy.

To be clear: I don't think the old column is populated at all on prod.

Reply:
I'd like this to be non-nullable eventually.